### ISSUE TITLE
Additional Damage not being parsing

### ISSUE THREAD

**Issue by Cavok7** [ravahn/BnS_ACT_Plugin#37]
Additional Damage from Fighting Spirit/Punishment buff/BT Earring is not being parsing.

Example:
[13:23:52.053] Blade Echo critically hit Green Training Dummy for 27,282 damage.
[13:23:52.345]  Green Training Dummy received 1987 damage from .

Changing `regex_incomingdamage1` to 
`Regex(@"(?<target>.+?)?( received|Received) (?<damage>\d+(,\d+)*) ((?<critical>Critical) )?damage((,)?( and)? (?<HPDrain>\d+(,\d+)*) HP drain)?((,)?( and)? (?<FocusDrain>\d+) Focus drain)?((,)?( and)? (?<debuff>.+?))? from ((?<actor>.+?)&apos;s )?(?<skill>.+?)?\.", RegexOptions.Compiled);` appears to fix the issue without breaking anything important.

`if (string.IsNullOrWhiteSpace(skill))` does not rename the skill however, as the current implementation renames the skill based on its actor name which is null.

Changing to this appears to allow renaming of the skill effect without breaking anything.

                    if (string.IsNullOrWhiteSpace(skill))
										
                     {
                         if (!string.IsNullOrWhiteSpace(actor))
                        {
                          //  "Received 1373 damage from Rising Blaze&apos;s ."
                             skill = actor;
                         }
                        if (string.IsNullOrWhiteSpace(actor))
                        { 
                          //  "received 1987 damage from ."
                             skill = "Additional Damage";
                         }
                      }

**Comment by xjzhang2037** [comment]
If someone is in your party also triggers additional damage.
Is the text for green dummy the same?

If so, wouldn't that mean that someone else's additional damage would be added to your score?

**Comment by Cavok7** [comment]
Yes, but that already happens with Raven weapon's additional damage proc, Emberstomp, Short Fuse, Ivy Poison, Time Bomb, and certain boss abilities.
